File: rumahsehat_flutter/lib/DTO/AppointmentDTO.dart
class AppointmentDTO {
  final DateTime waktuAwal;
  final String dokterId;
  final String pasienId;

  AppointmentDTO(this.waktuAwal, this.dokterId, this.pasienId);

  Map<String, dynamic> toJson() {
    return {'waktu_awal':waktuAwal.toIso8601String(), 'dokter_id': dokterId, 'pasien_id': pasienId};
  }
}

File: rumahsehatapi/src/main/java/apap/tugas/akhir/rumahsehatapi/models/DTO/AppointmentDTO.java
package apap.tugas.akhir.rumahsehatapi.models.DTO;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Setter
@Getter
public class AppointmentDTO {
    private LocalDateTime waktuAwal;
    private String dokterId;
    private String pasienId;
}
